porting/nimble/include/nimble/storage_port.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef _STORAGE_PORT_H
+#define _STORAGE_PORT_H
+
+#include <stdio.h>
+
+typedef enum {
+    READONLY,
+    READWRITE
+} open_mode_t;
+
```

Comment:
I think storage should be handle nimble host storage and not dedicated API

Reply:
I am sorry I couldn't understand this, can you please elaborate it a bit?